docs/tilemap.asciidoc

```diff
@@ -68,8 +80,19 @@ NOTE: In Elasticsearch releases 1.4.3 and later, this functionality requires you
 
 The availability of these options varies depending on the aggregation you choose.
 
-Select *view options* to change the following aspects of the table:
+Select the *Options* tab to change the following aspects of the chart:
 
 *Shaded Circle Markers*:: Displays the markers with different shades based on the metric aggregation's value.
 *Scaled Circle Markers*:: Scale the size of the markers based on the metric aggregation's value.
-*Desaturate Map*:: Desaturate the map's color in order to make the markers stand out more clearly.
\ No newline at end of file
+*Shaded Geohash Grid*:: Displays the rectangular cells of the geohash grid instead of circular markers, with different 
+shades based on the metric aggregation's value.
+*Desaturate map tiles*:: Desaturate the map's color in order to make the markers stand out more clearly.
```

Comment:
Heat maps are going into this release, will need docs on those. Lots of new options here. https://github.com/elastic/kibana/pull/3830